src/Image/Image.tsx

```diff
@@ -22,7 +22,7 @@ export const Image: React.FunctionComponent<ImageProps> = React.memo(
                 {!props.useImgTag && (
                     <div
                         id={props.id}
-                        className={"div-tag" + (props.className ? ` ${props.className}` : "")}
+                        className={`div-tag${props.onClick ? " link" : ""} ${props.className ? props.className : ""}`}
```

Comment:
This will leave an empty space if className is not passed. Here's how to fix it
```suggestion
                        className={`div-tag${props.onClick ? " link" : ""}${props.className ? ` ${props.className}` : ""}`}
```

Reply:
Oh, haven't given much thought about the className considering it's a preexisting prop. Nice catch.